### ISSUE TITLE
Pretraining of Image and text models

### ISSUE THREAD

**Issue by guillembraso** [muyangyi/SimSeg#9]
Hi! Thanks for your great work. I am trying to train your model. I noticed that your config files set `pretrained=true` for both text and image models. However, in your supplementary material you seem to say that they are trained from scratch. I'd like to ask which one should I follow. Thanks in advance!

**Comment by muyangyi** [comment]
Text and image encoders are trained from scratch, please set `pretrained=false` when pre-training.    
Currently, the `pretrained` parameter in the evaluation configuration files doesn't matter, because the encoder weights are overwritten by our checkpoints anyway.

**Comment by guillembraso** [comment]
Thanks for your reply! Since you don't plan on releasing the training code, could you please at least update the training configuration to have the correct values that you used? Currently, the weight decay and learning rate values in your config file also do not match the ones that you provided in the paper. Thanks!
